docs/ug/cli.md

```diff
@@ -56,6 +56,7 @@ The section below provides explanations for each of the flags.
 
 * Cannot be used with `--repos`. The `--repos` flag will take precedence over this flag.
 * If both `--repos` and `--config` are not specified, RepoSense looks for config files in the `./config` directory.
+* Config files must follow [this](#configFiles.html) format.
```

Comment:
Is the link pointing to the correct path? I think the `#` is for linking to sections on the current page instead of other pages.

Reply:
Thanks for pointing it out! Would an absolute path like this (https://reposense.org/ug/configFiles.html) work better?

Reply:
That might work on the production site, but may not be too ideal as it won't point to the corresponding page when deployed on different environments (e.g. localhost, or the surge preview deployments), and would also need to be changed if we change the domain in the future.

Markbind [has a feature](https://markbind.org/userGuide/formattingContents.html#intra-site-links) to get around this problem by specifying absolute paths as `{{ baseUrl }}/path`. Alternatively, it might be worth looking into if you can use relative paths for pages (i.e, removing the `#` in the link).